packages/itwinui-css/src/avatar/sizes.scss

```diff
@@ -9,6 +9,8 @@
 
   @if ($size == $iui-icons-xl) {
     --_iui-avatar-status-size: #{$iui-xs * 1.5};
+  } @else if ($size == $iui-icons-large) {
+    --_iui-avatar-status-size: #{$iui-xs};
```

Comment:
I kept the new `else if` for `$iui-icons-large` as that's something that still should be fixed, but I am no longer setting a variable for the bubble size and am instead using the `width/height: 25%` alongside `box-sizing: content=box`. This allows the bubble to change size dynamically for any size of avatar. I do think that at some point we should consider changing how the icon size itself is set so that it's also a percentage and able to change dynamically as well.